Pod/Classes/MWGridViewController.m

```diff
@@ -86,9 +88,12 @@ - (void)viewDidLayoutSubviews {
 - (void)adjustOffsetsAsRequired {
     
     // Move to previous content offset
-    if (_initialContentOffset.y != CGFLOAT_MAX) {
-        self.collectionView.contentOffset = _initialContentOffset;
-        [self.collectionView layoutIfNeeded]; // Layout after content offset change
+    if (@available(iOS 11.0, *)) {
```

Comment:
`!@available` geht ned?

Reply:
nein da sagt er "() is not convertible to bool"